src/server/sanity/Queries.ts

```diff
@@ -9,12 +9,14 @@ export const hentAvansertDokumentQuery = (
   _type == "block"=> {..., markDefs[]{
     ...,
     flettefeltReferanse->,
+    htmlfeltReferanse->,
     delmalReferanse->,
     valgReferanse->{..., valg[]{..., delmal->}}
     }
   },
   _type == "valgBlock" => {..., valgReferanse->{..., valg[]{..., delmal->}}},
   _type == "flettefelt" => {..., flettefeltReferanse->},
+  _type == "htmlfelt" => {..., htmlfeltReferanse->},
```

Comment:
Vi vet ikke helt om vi trenger begge disse, men vi trengte de for å kunne bruke `return felt ? felt : htmlfeltReferanse.felt;`
Uten de så kom ikke felt med

Reply:
Trooor vi burde ha begge 🤔